wfc_dylib/src/lib.rs

```diff
@@ -434,6 +465,9 @@ pub enum WfcObserveResult {
 pub extern "C" fn wfc_observe(
     wfc_world_state_handle: WfcWorldStateHandle,
     wfc_rng_state_handle: WfcRngStateHandle,
+    limit_observations_usize: usize,
```

Comment:
Is this a bool in disguise? If so, pack it together with max_observations. 0 can mean unlimited, if you restructure the code slightly.

Reply:
I know but it seemed to be cryptic. I can change it though.

Reply:
Please do. It might seem cryptic to you, but this is a very common practice in certain circles, to whom your solution would be more cryptic.

Reply:
Done. this piece of code is outdated.